fe/fe-core/src/main/java/com/starrocks/sql/ast/BoundaryLiteral.java

```diff
@@ -0,0 +1,45 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+package com.starrocks.sql.ast;
+
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.LiteralExpr;
+import com.starrocks.sql.common.ErrorType;
+import com.starrocks.sql.common.StarRocksPlannerException;
+import com.starrocks.thrift.TExprNode;
+
+public class BoundaryLiteral extends LiteralExpr {
```

Comment:
It's unnecessary?  why don't use stringliteral?

Reply:
> It's unnecessary? why don't use stringliteral?

Have remove this, just use unitBoundary for (STAET | END) now.